src/main/resources/hudson/plugins/nextexecutions/NextExecutionsWidget/index.jelly

```diff
@@ -26,9 +26,9 @@
       <tr>
         <td><a tooltip="${w.name}" href="${w.url}">${w.name}</a></td>
         <td tooltip="${w.timeToGo}">${w.date}</td>
+        <td tooltip="${w.

```

Comment:
Something's missing here? :-)

Reply:
You're right. I'll fix this when I get home. Don't know what I was smoking 😶.